src/main/java/it/gov/pagopa/pu/organization/connector/taxonomy/TaxonomyService.java

```diff
@@ -0,0 +1,9 @@
+package it.gov.pagopa.pu.organization.connector.taxonomy;
```

Comment:
let's use ms name as package name
```suggestion
package it.gov.pagopa.pu.organization.connector.pagopapayments;
```